src/github.com/matrix-org/dendrite/roomserver/storage/sql.go

```diff
@@ -307,17 +337,24 @@ CREATE TABLE IF NOT EXISTS events (
     event_id TEXT NOT NULL CONSTRAINT event_id_unique UNIQUE,
     -- The sha256 reference hash for the event.
     -- Needed for setting reference hashes when sending new events.
-    reference_sha256 BYTEA NOT NULL
+    reference_sha256 BYTEA NOT NULL,
+    -- A list of numeric IDs for events that can authenticate this event.
```

Comment:
> numeric IDs for events

You mean:

> numeric event IDs

As in `Event.EventNID` on https://github.com/matrix-org/dendrite/pull/4/files#diff-991257c04f932031b519c68ccb3ebdcaR58 ?